src/nvim/testdir/test_autocmd.vim

```diff
@@ -372,7 +372,7 @@ func Test_autocmd_bufwipe_in_SessLoadPost()
 	\ 'au VimLeave * call WriteErrors()',
         \ ]
   call writefile(content, 'Xvimrc')
-  call system(v:progpath. ' --headless -i NONE -u Xvimrc --noplugins -S Session.vim -c cq')
+  call system(expand(v:progpath). ' --headless -i NONE -u Xvimrc --noplugins -S Session.vim -c cq')
```

Comment:
Already changes this line to add `--headless` so adding another change is fine.

Though I would like to eventually change Nvim so that `-es` works without `--headless`.

Reply:
Sure for `set shellslash` and `set shell=sh` (or bash, wsl).